addons/binding/org.openhab.binding.co2signal/src/main/java/org/openhab/binding/co2signal/internal/json/.project

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There should be no .project file deep in your source tree, please remove.

Reply:
Removed